packages/wallets/src/features/cfd/types.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+    useAvailableMT5Accounts,
+    useCreateOtherCFDAccount,
+    useCtraderAccountsList,
+    useDxtradeAccountsList,
+    useMT5AccountsList,
+    useSortedMT5Accounts,
+} from '@deriv/api';
+
+// eslint-disable-next-line  @typescript-eslint/no-namespace
+export namespace THooks {
+    export type AvailableMT5Accounts = NonNullable<ReturnType<typeof useAvailableMT5Accounts>['data']>[number];
+    export type CtraderAccountsList = NonNullable<ReturnType<typeof useCtraderAccountsList>['data']>[number];
+    export type DxtradeAccountsList = NonNullable<ReturnType<typeof useDxtradeAccountsList>['data']>[number];
+    export type MT5AccountsList = NonNullable<ReturnType<typeof useMT5AccountsList>['data']>[number];
+    export type SortedMT5Accounts = NonNullable<ReturnType<typeof useSortedMT5Accounts>['data']>[number];
```

Comment:
suggestion/idea for future: wouldn't it be simpler to base types on DDD?
e.g. just have Accounts[] or MT5Accounts[] ?

Current patterns derive types from hooks, which means, separate type for every hook, even if they return array of accounts.